src/app/components/Convert/Convert.tsx

```diff
@@ -8,19 +8,21 @@ import useTranslation from 'app/hooks/useTranslation';
 
 export const Convert = () => {
   const {
-    showResume,
-    isConvertExpired,
     currencies,
-    handleSubmit,
-    onSubmit,
-    register,
+    error,
     errors,
+    expiryTime,
     getConvertData,
     handleConfirm,
+    handleSubmit,
+    isConfirmConvertBalanceSuccess,
+    isConvertBalanceSuccess,
+    isConvertExpired,
+    isFirstConversion,
     isLoading,
-    error,
+    onSubmit,
+    register,
   } = useConvert();
-
   const t = useTranslation();
 
```

Comment:
can we return CustomLoader here when isLoading? to avoid ternary and fragments below